p2p/src/store.rs

```diff
@@ -94,11 +95,8 @@ impl PeerStore {
 	}
 
 	pub fn save_peer(&self, p: &PeerData) -> Result<(), Error> {
-		// we want to ignore any peer without a well-defined ip
```

Comment:
I *think* this is safe to remove now that we are correctly extracting ip addresses from the connection.
This was making testing locally really difficult as we were not saving anything with 127.0.0.1 ip address (even if we were testing locally where this was actually desired).